build.rs

```diff
@@ -48,6 +48,9 @@ fn main() {
     if let Some(rev) = git_revision_hash() {
         println!("cargo:rustc-env=RIPGREP_BUILD_GIT_HASH={}", rev);
     }
+
+    // embed windows application manifest file
```

Comment:
From what I see browsing through the code base, comments are written as sentences, i.e. they start with a capital letter and end with a period.

Reply:
Yeah, that's preferred. But what would be better here is more elaboration. Mostly, this comment nearly repeats what the code itself says, without adding much. I think this comment should:

* Say why this is being done.
* Include instruction on how to take advantage of it. If you're someone who is reading this `build.rs` trying to understand it, and we say, "this enables long file paths" without saying that it also requires a registry edit, that might produce some confusion. So let's shine a light where we can.